### PR TITLE
added quiz app screens

### PR DESCRIPTION
**Pull request by anasm11**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,60 @@
...
+                    5 questions
+                </p>
+                </div>
+            </div>
+        </div>
+
+        <div class='card-container'>
+      <div class="card basic">
+        <div class="card-body">
+          <img src="https://picsum.photos/400/300" />
```
Comment (Mohitk391) [inline index.html]: Suggestion: you should add alt here..

File: login.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+    padding:1rem;
+    gap:1rem;
+}
+
+.form-header{
+    color:var(--primary-color);
+    margin:0;
```
Comment (Mohitk391) [inline login.css]: Suggestion: Is this needed??i think it is 0 by default so you can remove this...
One thing you can do is reset margin at the beginning of styling...

File: questionpage.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
+body{
+    width:600px;
```
Comment (Mohitk391) [inline questionpage.css:2]: Suggestion:  Can you convert this into rem/em..??for better responsiveness

File: result.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
+body{
+    width:600px;
```
Comment (Mohitk391) [inline result.css:2]: Same here...convert into rem/em..

File: rules.css [hunk 5]
```diff
@@ -0,0 +1,9 @@
+body{
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+    width:600px;
```
Comment (Mohitk391) [inline rules.css:5]: Here as well...px to rem/em

File: signup.css [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+    padding:1rem;
+    gap:1rem;
+}
+
+.form-header{
+    color:var(--primary-color);
+    margin:0;
```
Comment (Mohitk391) [inline signup.css]: Suggestion: can u remove this..?

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    display:flex;
+    gap:2rem;
+}
+
+.card{
+    overflow:hidden;
+}
+
+.card-container{
+    width:275px;
```
Comment (Mohitk391) [inline styles.css:19]: Same here...px to rem/em

**Review by Mohitk391 — APPROVED** [review]
Appreciation: Apart from some suggestions, code looks clean, readable and UI looks great as well...Just take care of responsive code and you are good to go...you can merge this branch..

File: login.css [hunk 8]
```diff
@@ -0,0 +1,40 @@
+body{
+    background-color: #eeeeee;
```
Comment (yaswanthmyneni) [inline login.css]: changes: use css variables for colors

File: login.css [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+    height:90vh;
+    display:flex;
+    align-items:center;
+    justify-content:center;
+}
+
+.form{
+    background-color: white;
+    border-radius: 1rem;
+    box-shadow:0 0 0.2rem 0rem;
```
Comment (yaswanthmyneni) [inline login.css]: suggestion: use px for less than 16px everywhere

**Review by yaswanthmyneni — APPROVED** [review]
use CSS variables for colors everywhere, you can merge after changes

**Pull request merged**